### DISCUSSION TITLE
Looking for a team to start a new Awesome-Github-Profile-readme repository

### DISCUSSION THREAD

**Discussion by samuel-ma** [abhisheknaiidu/awesome-github-profile-readme#900] [General]
Hey, here's an idea
Whose ok with starting off a new repo for The-Most-Awesome-Github-Profile-Readmes
I know a great designer, let's just form a team
We can recruit on the community discussion's feed

**Comment by Prakash4844** [comment]
It might be a good idea, as it seems the author has abandoned this repository.
